hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ReopenTableRegionsProcedure.java

```diff
@@ -57,7 +58,7 @@
 
   // Specify specific regions of a table to reopen.
   // if specified null, all regions of the table will be reopened.
-  private final List<byte[]> regionNames;
+  private List<byte[]> regionNames;
```

Comment:
So here, null means reopen all the regions, and an empty regionNames means do not reopen any regions? Then I think the current serialize and deserialize will have trouble? You can not tell whether regionNames is null or empty after serializing...

Reply:
Basically, if `regionNames` is null or empty, then we will reopen all regions. Only if the list contains at least one regionName, we will reopen only those regions specified in list.

Corresponding logic is here https://github.com/apache/hbase/pull/2494/files#diff-26b8b57193686e9efa5abc24cd9b7c5bL174-L175:
```
    if (CollectionUtils.isNotEmpty(regionNames) &&
      CollectionUtils.isNotEmpty(tableRegionsForReopen)) {
```

Reply:
Then just use Collections.emptyList here to avoid null check since null and emptyList does not make any difference?

Reply:
Sounds good option. Let me update.
Thanks